src/globals/scss/objects/_form-group.scss

```diff
@@ -0,0 +1,21 @@
+@include exports("form-group") {
+
+  .govuk-o-form-group {
+    @include govuk-responsive-margin($govuk-spacing-responsive-6, "bottom");
+
+    .govuk-o-form-group:last-of-type {
```

Comment:
Still not sure about these resets in this file, I don't think having the additional nested classes is a great idea, since they seem to be a limitation of how we're doing our macros.

We could come back to this after this feature is merged though.

Reply:
I think this is a valid use case in case of the date-input for instance where you have outer and inner form-group. But happy to review this after.